contracts/impls/centralized/WitnetRequestBoard.sol

```diff
@@ -111,11 +111,13 @@ contract WitnetRequestBoard
         returns (bytes memory _bytecode)
     {
         WitnetTypes.DataRequest storage _dr = __dataRequest(id);
-        _bytecode = WitnetRequest(_dr.addr).bytecode();
-        require(
-            WitnetTypes.computeDataRequestCodehash(_bytecode) == _dr.codehash,
-            "WitnetRequestBoard: manipulated request"
-        );
+        if (_dr.addr != address(0)) {
```

Comment:
Let's add a comment explaining why this is needed:

The bridge node uses the `readDataRequest` function to check if the data request has been deleted. In that case, `readDataRequest` should return an empty array. This `if (_dr.addr != address(0))` is used to check the case where the data request has been deleted, because dr.addr is set to 0 in that case.